src/game/hooks/Anticheat/SendMetric.cpp

```diff
@@ -13,10 +15,16 @@ namespace YimMenu::Hooks
 {
 	bool Anticheat::SendMetric(void* manager, rage::rlMetric* metric)
 	{
+		const auto bad_metrics = std::unordered_set<std::string_view>({"PCSETTINGS", "CHEAT", "WEATHER", "FIRST_VEH", "HARDWARE_OS", "HARDWARE_CPU", "HARDWARE_GPU", "HARDWARE_MOBO", "HARDWARE_MEM", "CASH_CREATED"});
 		char buffer[256]{};
 		rage::rlJson serializer(buffer, sizeof(buffer));
 
 		metric->Serialize(&serializer);
+		if (bad_metrics.contains(metric->GetName()))
```

Comment:
Factor out metric->GetName() into a variable so that when the logger is fired, it doesn't have to call GetName() again (compiler won't optimize this out).